projects/rulesets-base/src/custom-rulesets/load-ruleset.ts

```diff
@@ -0,0 +1,5 @@
+// Handle ESM and CJS modules
+export function loadRuleset(path: string) {
```

Comment:
So we were using dynamic imports before which effectively got compiled down to this. 

This require + esm interop allowed us to handle both CJS + ESM ruleset files, and dynamic imports _only_ supported ESM.